Python/executor_cases.c.h

```diff
@@ -27,311 +27,312 @@
         }
 
         case _STORE_FAST_1: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 1;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST_2: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 2;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST_3: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 3;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST_4: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 4;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST_5: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 5;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST_6: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 6;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST_7: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = 7;
             assert(oparg == CURRENT_OPARG());
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _STORE_FAST: {
-            PyObject *value;
+            _Py_TaggedObject value;
             oparg = CURRENT_OPARG();
-            value = stack_pointer[-1];
+            value = (stack_pointer[-1]);
             SETLOCAL(oparg, value);
             stack_pointer += -1;
             break;
         }
 
         case _POP_TOP: {
-            PyObject *value;
-            value = stack_pointer[-1];
+            PyObject * value;
+            value = Py_CLEAR_TAG(stack_pointer[-1]);
```

Comment:
The generated code here for `DECREF_INPUTS()` doesn't seem correct: this unconditionally decref's the value, but we'll only want to decref it if it's not tagged.